homeassistant/components/uptimerobot/switch.py

```diff
@@ -62,7 +61,9 @@ async def _async_edit_monitor(self, **kwargs: Any) -> None:
             response = await self.api.async_edit_monitor(**kwargs)
             self.async_write_ha_state()
         except UptimeRobotAuthenticationException:
-            self._config_entry.async_start_reauth(self.hass)
+            if self.coordinator.config_entry:
```

Comment:
We do never not have the config_entry here, this integration can only be setup with config entry

Reply:
so you prefer a assert ?

Reply:
No, just access it.
If mypy complains  add it to the coordinator